{"uid":"a17883adb494eb4e","name":"User is able to login,this should fail","fullName":"tests.test_login.TestLogin#test_user_login","historyId":"39cff70f68670e41090de5b2f95773fe","time":{"start":1683400647727,"stop":1683400679118,"duration":31391},"description":"User is able to login","descriptionHtml":"<p>User is able to login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message:","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f28c2dbe410>\n\n    @allure.description(\"User is able to login\")\n    @allure.title(\"User is able to login,this should fail\")\n    def test_user_login(self):\n>       self.login_page.login_existing_user(\n            \"01116888790\", \"test\", \"alyaothman\")\n\ntests/test_login.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/login_page.py:54: in login_existing_user\n    self.fill_password(password)\npages/login_page.py:43: in fill_password\n    self.fill_text(self.PASSWORD, password)\npages/base_page.py:26: in fill_text\n    element: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"b17c55b761ac485567b9fa8730c02f98\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f28c2e66980>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../../.local/share/virtualenvs/selenium-python-tOOKI_rJ/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1683400645394,"stop":1683400647726,"duration":2332},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"User is able to login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message:","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f28c2dbe410>\n\n    @allure.description(\"User is able to login\")\n    @allure.title(\"User is able to login,this should fail\")\n    def test_user_login(self):\n>       self.login_page.login_existing_user(\n            \"01116888790\", \"test\", \"alyaothman\")\n\ntests/test_login.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/login_page.py:54: in login_existing_user\n    self.fill_password(password)\npages/login_page.py:43: in fill_password\n    self.fill_text(self.PASSWORD, password)\npages/base_page.py:26: in fill_text\n    element: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"b17c55b761ac485567b9fa8730c02f98\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f28c2e66980>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../../.local/share/virtualenvs/selenium-python-tOOKI_rJ/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"User logs in","time":{"start":1683400647727,"stop":1683400679117,"duration":31390},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \n\n","statusTrace":"  File \"/home/runner/.local/share/virtualenvs/selenium-python-tOOKI_rJ/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/selenium-python/selenium-python/pages/login_page.py\", line 54, in login_existing_user\n    self.fill_password(password)\n  File \"/home/runner/.local/share/virtualenvs/selenium-python-tOOKI_rJ/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/selenium-python/selenium-python/pages/login_page.py\", line 43, in fill_password\n    self.fill_text(self.PASSWORD, password)\n  File \"/home/runner/work/selenium-python/selenium-python/pages/base_page.py\", line 26, in fill_text\n    element: WebElement = self.wait.until(\n                          ^^^^^^^^^^^^^^^^\n  File \"/home/runner/.local/share/virtualenvs/selenium-python-tOOKI_rJ/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"User clicks login icon in nav bar","time":{"start":1683400647727,"stop":1683400648096,"duration":369},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Fill phone number","time":{"start":1683400648096,"stop":1683400648523,"duration":427},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"phone_number","value":"'01116888790'"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"User clicks start exploring","time":{"start":1683400648523,"stop":1683400648840,"duration":317},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"User fill password","time":{"start":1683400648840,"stop":1683400679117,"duration":30277},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \n\n","statusTrace":"  File \"/home/runner/.local/share/virtualenvs/selenium-python-tOOKI_rJ/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/selenium-python/selenium-python/pages/login_page.py\", line 43, in fill_password\n    self.fill_text(self.PASSWORD, password)\n  File \"/home/runner/work/selenium-python/selenium-python/pages/base_page.py\", line 26, in fill_text\n    element: WebElement = self.wait.until(\n                          ^^^^^^^^^^^^^^^^\n  File \"/home/runner/.local/share/virtualenvs/selenium-python-tOOKI_rJ/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"password","value":"'test'"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[{"name":"phone_number","value":"'01116888790'"},{"name":"password","value":"'test'"},{"name":"username","value":"'alyaothman'"}],"stepsCount":4,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":5,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"setup::0","time":{"start":1683400679125,"stop":1683400679239,"duration":114},"status":"passed","steps":[],"attachments":[{"uid":"c2de1d8df65f1e03","name":"Console Logs","source":"c2de1d8df65f1e03.json","type":"application/json","size":246},{"uid":"e65d023b32032861","name":"Screenshot","source":"e65d023b32032861.png","type":"image/png","size":45591}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":false,"hasContent":true}],"labels":[{"name":"tag","value":"chrome"},{"name":"story","value":"Login functionality"},{"name":"tag","value":"@pytest.mark.usefixtures('setup')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az358-779"},{"name":"thread","value":"1770-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser","value":"'chrome'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"7309ca0376a2b76e","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1683400613369,"stop":1683400645251,"duration":31882}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["chrome","@pytest.mark.usefixtures('setup')"]},"source":"a17883adb494eb4e.json","parameterValues":["'chrome'"]}